File: app/src/main/java/com/example/rafaelanastacioalves/moby/domain/entities/MainEntity.java
package com.example.rafaelanastacioalves.moby.domain.entities;


public class MainEntity {

    private String id;
    private String title;
    private String price;
    private String price_currency;
    private String image_url;

    public MainEntity(){
        super();
    }

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getImage_url() {
        return image_url;
    }

    public String getPrice() {
        return price;
    }
}

File: app/src/main/java/com/example/rafaelanastacioalves/moby/entities/MainEntity.kt
package com.example.rafaelanastacioalves.moby.entities;


class MainEntity() {

    private lateinit var id: String;
    private lateinit var title: String;
    private lateinit var price: String;
    private lateinit var price_currency: String;
    private lateinit var image_url: String;

    fun  getId(): String {
        return id;
    }

    fun getTitle(): String {
        return title;
    }

    fun getImage_url(): String {
        return image_url;
    }
}
